cdhweb/static_src/global/css-variables/typography.scss

```diff
@@ -1,7 +1,7 @@
 :root {
-  --font-family-primary: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
-    'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  --font-family-primary: 'Brown', -apple-system, BlinkMacSystemFont, 'Segoe UI',
```

Comment:
OK I'm bikeshedding here but the [Geometric humanist](https://github.com/system-fonts/modern-font-stacks?tab=readme-ov-file#geometric-humanist) set is probably a pretty good fit for Brown

Reply:
many thanks